packages/models-library/src/models_library/clusters.py

```diff
@@ -110,11 +110,23 @@ class Config:
 
 
 class Cluster(BaseCluster):
-    id: ClusterID = Field(..., description="The cluster ID")
+    id: Union[ClusterID, Literal["default"]] = Field(..., description="The cluster ID")
 
     class Config(BaseCluster.Config):
         schema_extra = {
             "examples": [
+                {
+                    "id": "default",
+                    "name": "The default cluster",
```

Comment:
Potentially: "The default cluster available to all users"

Reply:
well the cluster model already defines the access rights. I do not see the benefit of adding hard-coded string to describe it further. already writing "default" in the name is on the edge.